bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/thing/bridge-types.xml

```diff
@@ -1,14 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<thing:thing-descriptions bindingId="openweathermap"
```

Comment:
The reason it didn't work is that the `bindingId` is wrong in this file

Reply:
@Hilbrand OK.  That is the real problem then. My fix was more of a work around that worked.  

What is better, my fix or changing the bindingId?

Reply:
In general there is no best choice. Either solution is ok. In this case it probably makes more sense to have everything in one file as it's only a small set of things. I would say your fix is ok.

Reply:
Sorry for the copy-paste error, was my bad :-(

Reply:
@Wolfgang1966 no problem it gave me an easy fix (work around).